about.html

```diff
@@ -0,0 +1,77 @@
+        </ul>
+    </div>
+    <div class="desk-menu" id="deskMenu">
+        <ul class="desk-menu">
+          <li class="menu"><a class="desktop-menu" href="./index.html">Home</a></li>
+          <li class="menu"><a class="desktop-menu" href="#about.html">Program</a></li>
+          <li class="menu"><a class="desktop-menu" href="#contact-me">Join</a></li>
+          <li class="menu"><a class="desktop-menu" href="#about-me">Sponser</a></li>
+          <li class="menu"><a class="desktop-menu" href="#contact-me">Register</a></li>
+        </ul>
+    </div>
+   </div>
+    <div class="container">
+        <div class="abfront">
+            <div class="abfront1">
+            <h2>"hello! developers"</h2>
+            <h1>DEVELOPERS GLOBAL SUMMIT 2023</h1>
+            <p> <span> The Creative Commons Global Summit</span> , held every two years, is a place where CC activists from all over the
+                world and experts, scholars and activists from all walks of life who support and practice open culture gather together.</p>
+            </div>
+            <div class="abfront2">
+            <p class="abpbox1">In particular, at this year's CC Summit, we
+                are waiting for the participation of various organizations and individual participants who want to participate in activities to spread the value of sharing, free culture, and open knowledge with CC.
+                Free software supporters, Wikipedia activists, open knowledge foundation activists, art galleries, libraries, museums, archives, governments,
+                foundations, lawyers, social activists, etc. various people active in the field of open content and open knowledge gather in one place to
+                share richer and more dynamic We hope that it will be an opportunity to share our thoughts on creating an ecosystem.
+            </p>
+            <h5>June 2023 (Wednesday - Saturday) <br>
+                See you at the Global Summit</h5>
+            </div>
+        </div>
+    </div>
+        <div class="logowinner">
+            <h1>Logo for software developmet for 2023</h1>
+            <p class="redline">__________</p>
+            <p>The winner of the logo contest, which was held from June 8th to July 7th, was decided through online voting and evaluation by the judges.
+                After consultation with the winning designer, the logo of CC Global Summit 2015, which was optimized for use on the website and promotional materials, is as follows.</p>
+            <img src="images/africalogo.webp" alt="afrca logo images">
+        </div>  
+     
+     <div class="footer">
+        <p>The african software development</p>
+        <div class="logo">
+        <img src="images/africa-signal-logo-design-vector-600w-1454378303.webp" alt="logo images">
+        <img src="images/africalogo.webp" alt="logo images">
+       </div>
+        <img src="images/social media icons.png" alt="social media images">
+     </div>
```

Comment:
- [ ] Please change the alignment of contents on your design to match the design given. Please look at the comparison below 👇 
Your design
![image](https://user-images.githubusercontent.com/90222110/198279118-6e5da070-df2d-46ce-b554-951f2763939c.png)
**Given design**
![image](https://user-images.githubusercontent.com/90222110/198279172-82c51788-41d4-4b54-8a9b-811086f62ec7.png)

**NOTE** The background color is dark-grey, not black 😉